docs/resources/infoblox_ipv4_network_container.md

```diff
@@ -8,11 +8,15 @@ resource block:
 
 * `network_view`: optional, specifies the network view in which to create the network container; if a value is not specified, the name `default` is used as the network view.
 * `cidr`: required, specifies the network block to use for the network container; do not use an IPv6 CIDR for an IPv4 network container.
```

Comment:
"required" - it is required only if `parent_cidr` is not specified. Please add a description of relations between the fields.